src/Serialization/Fields.jl

```diff
@@ -48,3 +48,68 @@ function load_internal(s::DeserializerState, ::Type{gfp_fmpz_elem}, dict::Dict)
     F = load_type_dispatch(s, Nemo.GaloisFmpzField, dict[:parent])
     return F(load_type_dispatch(s, fmpz, dict[:data]))
 end
+
+################################################################################
+# AnticNumberfield
+function save_internal(s::SerializerState, K::AnticNumberField)
+    return Dict(
+        :def_pol => save_type_dispatch(s, defining_polynomial(K))
+    )
+end
+
+function load_internal(s::DeserializerState, ::Type{AnticNumberField}, dict::Dict)
+    def_pol = load_type_dispatch(s, dict[:def_pol], check_namespace=false)
+    K, _ = NumberField(def_pol)
+    return K
+end
+
+#elements
+function save_internal(s::SerializerState, k::nf_elem)
+    K = parent(k)
+    polynomial = parent(defining_polynomial(K))(k)
+    K_dict = save_type_dispatch(s, K)
+    
+    return Dict(
+        :parent => K_dict,
+        :polynomial => save_type_dispatch(s, polynomial)
+    )
+end
+
+function load_internal(s::DeserializerState, ::Type{nf_elem}, dict::Dict)
+    K = load_type_dispatch(s, dict[:parent], check_namespace=false)
```

Comment:
OK, more complicated: this cannot work in general. You rely here on using `cached= true` in load to ensure that the same field is used for all elements (if you have more than one). Basically all parent objetcs (rings, groups, ...) have to be handled specially: they can only be created once and have to use xref for all other instances. Otherwise this cannot work: consider the field defined by `x^6+108` - which is the splitting field of `x^3-2`, so the degree 6 field has three cubic subfields: `Q(2^(1/3))`, `Q(2^(1/3) z_3)` and `Q(2^(1/3) z_3^2)` all given via the same polynomial `x^3-2`. Clearly as subsets of the complex numbers, they are distinct. So to treat them seperately, I have to create the number field with `cached = false`. If my data involves elements you see the problem...

Reply:
So if I am not mistaken this is more an issue on saving elements that come from different but isomorphic fields, however if these elements arise in the same polynomial then it should be alright? but maybe there is still an issue on loading a polynomial since we might have that on load 2^(1/3) z_3 becomes 2^(1/3) ?